### ISSUE TITLE
Scrollbar incorrectly bottoms out at fixed number of lines

### ISSUE THREAD

**Issue by maxbrunsfeld** [atom/atom#13459]
We render a scrollbar by creating a div with a height that reflects the editor's screen line count. Unfortunately DOM nodes' heights seem to be bounded:

![max-height](https://cloud.githubusercontent.com/assets/326587/21280897/adccf898-c39d-11e6-8e62-fb6b682ed2c2.png)

Reported by @ungb and @Ben3eeE 

/cc @nathansobo @as-cii Any ideas on how to solve this?

**Comment by nathansobo** [comment]
We could render the scrollbar ourselves but I'm not keen to deal with that.

**Comment by maxbrunsfeld** [comment]
Yeah, I agree. Maybe we could shrink the scrollbar view and its `scrollTop` proportionally, in order to keep its scroll position correct. It would distort the *size* of the scrollbar itself (making it taller than it would otherwise be), but that might not be so bad.

**Comment by nathansobo** [comment]
Great idea about doing it proportional. That's the way to go.
